.github/workflows/pull-request.yaml

```diff
@@ -16,6 +16,7 @@ on:
     - synchronize
 permissions:
   contents: read
+  pull-requests: write
```

Comment:
@eduardo-getpassport At the top we should keep the permissions minimal and just grant the `write` permission to the `comment-pr` job.